docs/index.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Introduction
+nav_order: 1
+---
+
+# :small_red_triangle: T-Systems MMS Phonebook :small_red_triangle:
+
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)](https://github.com/semantic-release/semantic-release) [![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/) [![GitHub issues](https://img.shields.io/github/issues/T-Systems-MMS/phonebook.svg?style=popout)](https://github.com/T-Systems-MMS/phonebook/issues) [![GitHub](https://img.shields.io/github/license/T-Systems-MMS/phonebook.svg)](https://github.com/T-Systems-MMS/phonebook/blob/master/LICENSE)
+
+  
+
+## **:question: What is the Phonebook?**
+
+The Phonebook is a community project for us, from us and everybody is welcome to participate.
+This Single-Page-Application is able to display People, Rooms and the organization structure. It is perfect to search for people in a company, get information where they sit in your building and how to contact them.
+
+## **:wave: New to the Phonebook?**
+We are always searching for new contributors who want's to be a part of a growing community. The contribution beneftis are huge. You can learn many about the below technologies, bring new ideas into the project, link up with new people from different departments and learn how it is to be an OpenSource-Developer.
+
+If you're just getting started with the Phonebook, we recommend you to take a look on our [getting-started](/docs/getting-started) page or take a look at our [Project Board](https://github.com/T-Systems-MMS/phonebook/projects/1).
+
+> *If you're a part of T-Systems you can also take a look at the internal get-started page.*
+
+## **:mag: What are we searching?**

```

Comment:
Irgendwie ergibt die Überschrift keinen Sinn mit dem darunter? Was wolltest du ausdrücken?

Reply:
You want to be a part of this community?